lib/converters/html-converter.js

```diff
@@ -32,12 +35,20 @@ module.exports = class HtmlConverter extends Converter {
    * @param data - The JSON data that is used by the engine to substitute variables.
    * @returns {Promise<void>}
    */
-  static convert(htmlInfo, data) {
+  convert(htmlInfo, data) {
     return new Promise(async (resolve, reject) => {
       let fileContent;
 
       try {
-        fileContent = await fs.readFile(htmlInfo.file, 'utf8');
+        if (this.logger) {
+          this.logger.debug(`Reading file "${htmlInfo.file}" from disk.`);
+        }
+
+        fileContent = fs.readFileSync(htmlInfo.file, 'utf8'); // For some reason the async version take a super long time.
```

Comment:
Perhaps node is scheduling other tasks in between. I see your new #87, so this would be a temp. workaround.